src/components/Dashboard/TodoList.vue

```diff
@@ -619,7 +624,8 @@ export default {
           draftTitle: 'Incorporation Application',
           status: filing.header.status,
           enabled: Boolean(task.enabled),
-          order: task.order
+          order: task.order,
+          paymentToken: filing.header.paymentToken || null
```

Comment:
This was a bug for resume/retry payment. The pay url was incorrect because it needs the payment token id.